src/vte.c

```diff
@@ -268,10 +268,11 @@ static void create_vte(void)
 	vc->menu = vte_create_popup_menu();
 	g_object_ref_sink(vc->menu);
 
-	frame = gtk_frame_new(NULL);
+	viewport = gtk_viewport_new (NULL, NULL);
+	gtk_viewport_set_shadow_type(GTK_VIEWPORT(viewport), GTK_SHADOW_NONE);
```

Comment:
> Well, I don't get it: if you want nothing, just remove the intermediate container and it should work, wouldn't it?

You are right, in this case it can be removed completely. Will fix that.

>But you can also set the shadow to NONE with a frame

As I said, shadow is already NONE with frame by default. But frame != shadow and the frame from GtkFrane cannot be removed as far as I know.